package.json

```diff
@@ -20,7 +20,8 @@
     "test:dev": "run-s test:dev:*",
     "test:ci": "run-s test:ci:*",
     "test:dev:ava": "ava",
-    "test:ci:ava": "c8 -r lcovonly -r text -r json ava",
+    "test:measure": "node tools/tests_duration.mjs",
+    "test:ci:ava": "c8 -r lcovonly -r text -r json ava --concurrency 1",
```

Comment:
I used `concurrency` as I was getting out of memory errors in CI.

AVA itself runs using `serial` in their CI, so we should consider doing the same, see https://github.com/avajs/ava/blob/ada1a4f8876ad7b2476440609691e301a02f3951/test/snapshot-workflow/README.md#serial-execution

- `concurrency` - number of parallel processes to spawn (a process is spawned per test file)
- `serial` - number of parallel tests within the same file to run